chunk/download.go

```diff
@@ -92,12 +92,14 @@ func downloadFromAPI(client *http.Client, request *Request, buffer []byte, delay
 		time.Sleep(time.Duration(delay) * time.Second)
 	}
 
+	if request.acknowledgeAbuse {
+		request.object.DownloadURL += "&acknowledgeAbuse=true"
+	}
```

Comment:
Please work with a copy of `DownloadURL`, you are modifying the request struct, that was passed into the function as a parameter.

```go
downloadURL := request.object.DownloadURL
if request.acknowledgeAbuse {
        downloadURL += "&acknowledgeAbuse=true"
}
req, err := http.NewRequest("GET", downloadURL, nil)
```

Reply:
Apologies for misunderstanding your initial request.  Implemented in 287938feea0b1246bea1977dc59e9f2ef003c4d6